src/components/utils/src/sqlite_wrapper/sql_database.cc

```diff
@@ -39,7 +39,7 @@ namespace dbms {
 const std::string SQLDatabase::kInMemory = ":memory:";
 const std::string SQLDatabase::kExtension = ".sqlite";
 
-SQLDatabase::SQLDatabase()
+SQLDatabase::SQLDatabase(Persistent)
```

Comment:
@BSolonenko lets rename typename to ‘Placement’. Please discuss with @mkorniichuk @AByzhynar @AKalinich-Luxoft

Reply:
@LuxoftAKutsan fixed in https://github.com/Ford-Luxoft/sdl_core/pull/17/commits/4154b9b3932adce8017f71b9fa54d0da8aa9e499